### PR TITLE
fix number formatting for 999 >= n > 99

### PR DESCRIPTION
**Pull request by dkent600**
Resolves:  https://daostack.tpondemand.com/restui/board.aspx?#page=board/5209716961861964288&appConfig=eyJhY2lkIjoiQjgzMTMzNDczNzlCMUI5QUE0RUE1NUVEOUQyQzdFNkIifQ==&boardPopup=bug/1559/silent

Was failing for numbers between 100 and 999.

See also:  https://github.com/daostack/alchemy/issues/1103

### CODE REVIEW

File: src/lib/util.ts [hunk 1]
```diff
@@ -133,9 +133,9 @@ export function formatTokens(amountWei: BN, symbol?: string, decimals = 18): str
     significantDigits = 1000000000;
     units = "B";
   }
-  else if (tokenAmount.ltn(1000)) {
+  else if (tokenAmount.ltn(100000)) {
```
Comment (tibetsprague) [inline src/lib/util.ts]: Doesn't this need to be a BN too?

Comment (dkent600) [reply src/lib/util.ts]: `tokenAmount` _is_ a BN

Comment (tibetsprague) [reply src/lib/util.ts]: Yes but 100000 is not

Comment (dkent600) [reply src/lib/util.ts]: The 1000000 doesn't have to be a BN.  It fits just fine in a `number` and `.ltn()`. takes a number by definition.  The function does have its own smaller limit on the size of the param, but throws an exception if that number is too big, which it isn't.

Comment (dkent600) [reply src/lib/util.ts]: The proof is that the unit tests succeed  :-)

Comment (tibetsprague) [reply src/lib/util.ts]: ok fine, but then why did you change the other one to new BN(...)?

Comment (dkent600) [reply src/lib/util.ts]: because that one exceeded the maximum value that `ltn()` allows for the input value.

**Pull request merged**